frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -324,6 +325,85 @@
             });
         };
 
+// Delete only a particular member
+vm.confirmMemberDelete = function(ev, participantTeamId, participantId) {
+    ev.stopPropagation();
+    // Appending dialog to document.body to cover sidenav in docs app
+    var confirm = $mdDialog.confirm()
+        .title('Would you like to remove this member?')
+        .textContent('Note: This action will remove this member from the team.')
```

Comment:
change this Note: `This action will remove this member from the team.` to `Note: This action will remove the member from team.`